### DISCUSSION TITLE
Extract the iOS modal into a separate module

### DISCUSSION THREAD

**Discussion by mmazzarolo** [mmazzarolo/react-native-modal-datetime-picker#600] [Ideas]
Following #318 we should:

1. Extract the picker sheet "wrapper" modal in its own module (e.g. `PickerSheetIOS`).
2. Integrate `PickerSheetIOS` in `react-native-modal-datetime-picker`
3. Optional: Create a cross-platform PickerModalDialogWhatever that exposes the same APIs on both Android and iOS.

**Comment by mmazzarolo** [comment]
FYI @eightyfive this is still pending, want to release v8 first

**Comment by eightyfive** [comment]
Hey, thanks for the heads up.

Sheet wise, I took another direction since I wanted to be as close as possible to the native HTML `<select />` feel when on Safari (iOS).

Hence I did put together this small [package](https://github.com/eightyfive/react-native-pick):

- [index.ios.js](https://github.com/eightyfive/react-native-pick/blob/master/index.ios.js)
- [picker-modal.ios.js](https://github.com/eightyfive/react-native-pick/blob/master/picker-modal.ios.js)
- [modal.ios.js](https://github.com/eightyfive/react-native-pick/blob/master/modal.ios.js)

**Comment by mmazzarolo** [comment]
Well, I still don't have enough time to work on it :/
